### PR TITLE
docs(process): add github refining document

### PR DESCRIPTION
**Pull request by marcod1419**
## Overview

This is a PR to add the GitHub Refining (previously known as GitHub Grooming) process to the Reference Architecture.

**Forum Issue #** https://github.com/telus/technology-forum/issues/199

---

#### Meta

> Please read and confirm each of the following:

- [x] this topic was discussed in the [Technology Forum][technology-forum] _(ignore if the pull request represents small changes)_
- [x] provided a descriptive topic and overview of contribution
- [x] documentation format follows the [topic template][template]
- [x] fork is up to date _(Hint: ["Syncing a Fork"][guide-forks])_
- [x] "work in progress" commits are squashed _(Hint: ["Squashing Commits"][guide-squash])_
- [x] commits follow the [Conventional ChangeLog][conventional-changelog] format
- [x] no sensitive content included, such as:
  - content considered competitive intelligence
  - security & privacy policy violating content
  - keys, tokens or credentials

[template]: https://github.com/telus/reference-architecture/blob/master/.template.md
[conventional-changelog]: https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular
[guide-forks]: https://help.github.com/articles/syncing-a-fork/
[guide-squash]: https://git-scm.com/book/id/v2/Git-Tools-Rewriting-History
[technology-forum]: https://github.com/telus/technology-forum]

### CODE REVIEW

File: process/github-refining.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+Every week, it is recommended that a new Github Refining Facilitator be selected. This person will be responsible for triaging issues on days where GitHub Refining is not happening. Additionally, they will lead the GitHub Refining meeting for that timeframe.
+
+### Process
+
+1. Review stories with the ‘triage’ or ‘review needed’ label, assess and assign an appropriate label, and rename the issue if unclear. _Every issue requires ‘status’, ‘type’, and ‘priority’ labels._
+  - First review triaged or unlabeled issues.
+  - Second, review issues that were marked as “Review Needed”
+
+2. Review Pull Requests
+  - Look at pull requests (oldest first) _not made by renovate or a team member_ and assign them to a team member to look at right away.
```
Comment (theetrain) [inline process/github-refining.md]: Update

> Look at pull requests (oldest first) ~not made by renovate or a team member~ `that is not in your internal backlog` and assign them to a team member...

**Comment by ruxandrafed** [comment]
@theetrain @marcod1419 is this good to go? :) I think we're fine, but there was a suggestion for a change from Enrico and then we just ran behind master ..

**Comment by mattcan** [comment]
@marcod1419 @theetrain I know it has been awhile. Are we ok to merge this?

**Comment by crimeminister** [comment]
A fun addition to this might be to include a snippet of a GitHub [Probot settings file](https://github.com/probot/settings) that defines this collection of labels.

**Pull request closed without merging**